[2022-03-20 14:04:23,374] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2020-01-05T00:00:00+00:00 [queued]>
[2022-03-20 14:04:23,404] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2020-01-05T00:00:00+00:00 [queued]>
[2022-03-20 14:04:23,415] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 14:04:23,415] {taskinstance.py:1068} INFO - Starting attempt 2 of 2
[2022-03-20 14:04:23,416] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 14:04:23,529] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): processing_user> on 2020-01-05T00:00:00+00:00
[2022-03-20 14:04:23,546] {standard_task_runner.py:52} INFO - Started process 99015 to run task
[2022-03-20 14:04:23,623] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'processing_user', '2020-01-05T00:00:00+00:00', '--job-id', '927', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpd0ovtfz5', '--error-file', '/tmp/tmp8lagg1s8']
[2022-03-20 14:04:23,630] {standard_task_runner.py:77} INFO - Job 927: Subtask processing_user
[2022-03-20 14:04:24,140] {logging_mixin.py:104} INFO - Running <TaskInstance: user_processing.processing_user 2020-01-05T00:00:00+00:00 [running]> on host airflowvm
[2022-03-20 14:04:24,332] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=processing_user
AIRFLOW_CTX_EXECUTION_DATE=2020-01-05T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-05T00:00:00+00:00
[2022-03-20 14:04:24,403] {python.py:151} INFO - Done. Returned value was: None
[2022-03-20 14:04:24,417] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=processing_user, execution_date=20200105T000000, start_date=20220320T140423, end_date=20220320T140424
[2022-03-20 14:04:24,856] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-03-20 14:04:24,883] {local_task_job.py:151} INFO - Task exited with return code 0
